src/js/structs/NodesList.js

```diff
@@ -27,7 +25,7 @@ module.exports = class NodesList extends List {
 
       // Marathon API
       if (filters.ids) {
-        hosts = _.filter(hosts, function (host) {
+        hosts = hosts.filter(function (host) {
           return this.ids.indexOf(host.id) !== -1;
         }, {ids: filters.ids});
```

Comment:
Hmmm... this works, but is used in a weird way in my opinion. We are setting the context to be `{ids: filters.ids}` and then `this.ids` refers to the `filters.ids`, but wouldn't it be more elegant to just refer to `filters.ids`, or even just create a variable `ids` outside of the callback?
I know this was not introduced by you, so we can do this in a separate PR, but I do think we need to change it